Gemfile.lock

```diff
@@ -322,10 +321,12 @@ DEPENDENCIES
   simplecov-rcov (~> 0.2.3)
   slimmer (= 8.2.1)
   smartdown (~> 0.15.1)
-  syck (= 1.0.5)
   therubyracer (~> 0.12.1)
   tilt (= 1.4.1)
   timecop
   uglifier
   unicorn (= 4.8.3)
   webmock (= 1.20.4)
+
+BUNDLED WITH
+   1.10.5
```

Comment:
I just installed ruby 2.2.2 locally and got the newest bundler version with it. It seems that bundler introduced this feature as they're planning to introduce breaking changes in version 2. This number is only increasing, never decreasing, so there shouldn't be too many issues. 

I think we have a few options here:

- roll with whatever the bundler team thinks is the best
- patch local bundler versions to avoid adding those lines
- remove it from this commit and open a separate PR to discuss what we want to do
- use pre 1.10 version of bundler locally